### PR TITLE
Refactor method delete_activity

### PR DESCRIPTION
**Pull request by luchiago**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Related Tickets & Documents
#3739 

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
Closing and reopening only to trigger Travis as it seems its manual interface is stuck

File: app/services/users/delete_activity.rb [hunk 1]
```diff
@@ -5,41 +5,26 @@ module DeleteActivity
...
-      user.follows.delete_all
-      Follow.followable_user(user.id).delete_all
-      user.messages.delete_all
-      Users::CleanupChatChannels.call(user)
-      user.mentions.delete_all
-      user.badge_achievements.delete_all
-      user.github_repos.delete_all
-
+      handle_social_media(user)
+      handle_profile_info(user)
```
Comment (Ridhwana) [inline app/services/users/delete_activity.rb]: is a more descriptive function name not `delete_profile_info` instead of `handle_profile_info`?

Comment (Ridhwana) [reply app/services/users/delete_activity.rb]: and the same for social media?

Comment (luchiago) [reply app/services/users/delete_activity.rb]: @Ridhwana I agree with you. I will change

Comment (Ridhwana) [reply app/services/users/delete_activity.rb]: Thanks @luchiago  😊 Once that's done this looks really good! 🎉 Thanks.

**Pull request merged**